README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
```

Comment:
Sorry, shouldn't this be just "Use the up arrow keys to rotate the shape."? I tried the game in my laptop and the down arrow key didn't rotate the shape, just accelerated its fall. Is that an error?